core/src/main/scala/org/apache/spark/sql/delta/commands/ConvertToDeltaCommand.scala

```diff
@@ -65,11 +67,13 @@ import org.apache.spark.util.SerializableConfiguration
  *
  * @param tableIdentifier the target parquet table.
  * @param partitionSchema the partition schema of the table, required when table is partitioned.
+ * @param collectStats Should collect column stats per file on convert.
  * @param deltaPath if provided, the delta log will be written to this location.
  */
 abstract class ConvertToDeltaCommandBase(
     tableIdentifier: TableIdentifier,
     partitionSchema: Option[StructType],
+    collectStats: Boolean = true,
```

Comment:
I have no idea how this is compiling, params with default values must go at the end of the param list.

anyways, please remove this default.

Reply:
also, please resolve conversations once you have addressed the comment :) this will make viewing the PR much easier.